frontend/src/routes/+layout.svelte

```diff
@@ -1,10 +1,17 @@
 <script lang="ts">
+	import { browser } from '$app/environment';
 	import { featureFlags } from '$lib/stores';
 	import { zenoFeatureFlags } from '$lib/util/features';
+	import * as amplitude from '@amplitude/analytics-browser';
 	import '../app.css';
 
 	export let data;
 	featureFlags.set({ ...zenoFeatureFlags, ...data.features });
+
+	if (browser)
+		amplitude.init('9e6755badfe6f970b509d3325a6a6678', {
```

Comment:
Should this be in here? I'd guess we might want different ones depending on dev/prod and also custom installations. Further, this should probably not be exposed, should it?

Reply:
re: dev vs. prod yeah maybe we want to read from ENV but changing ENV variables is a pain in prod aha.

It's okay to be exposed, it's usually just in the raw HTML.

Reply:
made it an env variable